src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+  let sum = 0;
+  const adder = (a = null) => {
+    if (a === null) {
+      sum = result;
```

Comment:
... and use just here: `const sum = result;`

So you need this `sum` only inside this block.